inst/extdata/OSD-error-reporting/RO/bozeman-hz.csv

```diff
@@ -42,6 +42,7 @@
 "FLEER","O1",5,0
 "FOLAVAR","Oi",5,0
 "GAPPMAYER","O1",5,0
+"GAPPMAYER","A21",NA,1554
```

Comment:
not sure where the `1554` is coming from here, but this A21 horizon was previously missing from the horizons parsed out of OSD

Reply:
some refinement of the OSD error reporting may be needed